src/openzeppelin/tests.cairo

```diff
@@ -1,7 +1,5 @@
 mod access;
-//mod test_accesscontrol;
```

Comment:
i fear this `mod access` might get confused by readers with the main access module instead of the test module. i'm not against the long list of `test_<module>.cairo` files. unsure that mimicking the file structure results in more clarity, but might be just me. @ericnordelo what do you think?

Reply:
Just for the record (I know we've discussed this already), I like the minimal grouping approach with just one directory of depth at most.